### PR TITLE
Update italian language

### PR DESCRIPTION
**Pull request by bovirus**
Complete italian translation
Use the feedback posted in PR 510.
I hope that I made all changes requested in PR 510.

### CODE REVIEW

**Comment by bovirus** [comment]
@schwabe 
@ordex 

I applied all (i hope..) step by step the suggestion prosed on PR 510.
Thanks.

File: res/openvpn-gui-res-it.rc [hunk 1]
```diff
@@ -234,25 +234,25 @@ FONT 8, "Microsoft Sans Serif"
 LANGUAGE LANG_ITALIAN, SUBLANG_DEFAULT
 BEGIN
     ICON ID_ICO_APP, ID_ICON_ABOUT, 8, 16, 21, 20
-    LTEXT "OpenVPN GUI v%ls - Interfaccia per Windows di OpenVPN\n\
+    LTEXT "OpenVPN GUI v%ls - Interfaccia di OpenVPN per Windows\n\
 Copyright (C) 2004-2005 Mathias Sundman <info@openvpn.se>\n\
 Copyright (C) 2008-2014 Heiko Hund <heikoh@users.sf.net>\n\
-Copyright (C) 2012-2022 OpenVPN GUI contributors\n", ID_TXT_VERSION, 36, 15, 206, 50
+Copyright (C) 2012-2021 OpenVPN GUI contributors\n", ID_TXT_VERSION, 36, 15, 206, 50
```
Comment (selvanair) [inline res/openvpn-gui-res-it.rc:240]: Leave 2022 as is

File: res/openvpn-gui-res-it.rc [hunk 2]
```diff
@@ -234,25 +234,25 @@ FONT 8, "Microsoft Sans Serif"
...
-
-    LTEXT "OpenVPN - Un'applicazione che permette di creare tunnel sicuri su reti IP \
-con una singola porta TCP/UDP, con il supporto basato sull'autenticazione e lo scambio \
-di chiavi SSL/TLS, cifratura, autenticazione e compressione dei pacchetti.\n\
+    LTEXT "OpenVPN - Un'applicazione che permette di creare tunnel sicuri \
+in reti IP  con una singola porta TCP/UDP, con il supporto basato \
+sull'autenticazione e lo scambio di chiavi SSL/TLS, cifratura, \
+autenticazione e compressione dei pacchetti.\n\
 \n", ID_LTEXT_ABOUT3, 8, 70, 240, 64
-    LTEXT "Copyright (C) 2002-2022 OpenVPN Technologies, Inc <info@openvpn.net>\n\
```
Comment (selvanair) [inline res/openvpn-gui-res-it.rc:247]: Here too.
You can add a fixup here -- please do not close and reopen the PR as that makes review hard. If its a minor thing like this one I can fix it too.

Thanks for the update, by the way :)

**Comment by bovirus** [comment]
@selvanair 

Could you please explain me what are the time plan to update this update?

**Comment by bovirus** [comment]
@selvanair

May I could do something?

**Comment by selvanair** [comment]
> @selvanair
> 
> May I could do something?

Any Italian speakers who could give this a quick review please? If this is an improvement and has no serious defects we can get this merged -- doesn't have to be perfect.

**Comment by bovirus** [comment]
@selvanair 

I see that is now availble a new beta for OpenVPN.
I hope to see the current italian language implemented soon.

**Comment by selvanair** [comment]
If no one objects in a day or so, I want to merge this, leaving further improvements up to future contributors. The last time @ordex had looked at an earlier version of this, the impression was that this brings in significant improvements though 
 some word/phrase choices were not ideal.

As for the merge conflicts, they are minor --- I can resolve them.

**Comment by selvanair** [comment]
@bovirus I've fixed the conflicts, combined your two commits into one and force pushed on to this PR. Also added a commit to fix some text truncation (see images below). Please check its okay.
| Text truncated  |  With fixup commit |
|------------------|----------------------|
|italian1|italian2|

**Comment by bovirus** [comment]
@selvanair 

Thanks for you acrtvities..

The fix is OK.

One time that you apply the changes I will start to translate current uintranslated strings.

Thnaks.

**Comment by bovirus** [comment]
@selvanair 

It seems that the italian fiel for GUi contains many strings untranslated.

This the italian language file with all strings translated

https://mega.nz/file/DUhAlIBY#RipULoOSQhB1jjK02aVVy7slb-OAtq6OiTmrEerZB3A

If you want I can apply it after you weill complete this current step.

**Comment by selvanair** [comment]
> @selvanair
> 
> It seems that the italian fiel for GUi contains many strings untranslated.
> 
> This the italian language file with all strings translated
> 
> https://mega.nz/file/DUhAlIBY#RipULoOSQhB1jjK02aVVy7slb-OAtq6OiTmrEerZB3A
> 
> If you want I can apply it after you weill complete this current step.

Please do not make any changes now. Will fix the text truncation and merge this later today. Then you can open a new PR for additional translations.

**Comment by bovirus** [comment]
@selvanair 

Absloutly agree. Infact I didn't make any changes.
I only advise that when you will post your changes there will be some others stes for me to do.

I'm waiting that you post your changes.
I will compare it with my translation and I will apply other changes in a new PR.

Thanks.

**Comment by selvanair** [comment]
@bovirus I fixed the text truncation (again) and reverted "Enable Pre-logon access provider" back to English so that you (or some else) can re-translate it properly.

**Review by selvanair — APPROVED** [review]

**Pull request merged**